patch/secure_boot_kernel_config.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Note: this script was created because there is a problem when changing the kernel config
+# values that are required by the Secure Boot feature when using patch/kconfig-inclusions (sonic flow to modify kernel flags).
+# So, when this problem will be resolved, this script should be removed and used the kconfig-inclusions.
+
+usage() {
+    cat <<EOF
+$0: # Display Help
+$0 <PEM_CERT>
+Script is modifying kernel config file to support system trusted key with custom certificate.
+Note: The signature algorithm used will be RSA over SHA512 x509 format.
+
+Parameters description:
+PEM_CERT                             public key (pem format). Key to be store in kernel.
+
+Usage example: bash secure_boot_kernel_config.sh cert.pem
+EOF
+}
+
+if [ "$1" = "-h" -o "$1" = "--help" ]; then 
+    usage
+fi
+
+echo "$0: Adding Secure Boot support in Kernel config file."
+
+CERT_PEM=$1
+
+[ -f "$CERT_PEM" ] || {
+    echo "Error: CERT_PEM file does not exist: $CERT_PEM"
+    usage
+    exit 1
+}
+
+local_cert_pem="debian/certs/$(basename $CERT_PEM)"
+linux_cfg_file="debian/build/build_amd64_none_amd64/.config"
+sed -i "s|^CONFIG_SYSTEM_TRUSTED_KEYS=.*|CONFIG_SYSTEM_TRUSTED_KEYS=\"$local_cert_pem\"|g" $linux_cfg_file
+sed -i 's/^CONFIG_MODULE_SIG_HASH=.*/CONFIG_MODULE_SIG_HASH="sha512"/g' $linux_cfg_file
+sed -i 's/^CONFIG_MODULE_SIG_SHA256=.*/# CONFIG_MODULE_SIG_SHA256 is not set/g' $linux_cfg_file
+sed -i 's/# CONFIG_MODULE_SIG_SHA512 is not set/CONFIG_MODULE_SIG_SHA512=y/g' $linux_cfg_file
+
+#lockdown feature disable
```

Comment:
Why does lockdown need to be disabled?

Reply:
We are aware of this point, from the Linux Debian definitions it can be enabled,  but when setting the Secure Boot enabled in SONiC with LockDown it created some conflict, we are working on it.